ajax/libs/ng-bootstrap/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "ng-bootstrap",
+  "filename": "ng-bootstrap.umd.min.js",
+  "version": "3.0.0",
+  "description": "Angular powered Bootstrap",
+  "keywords": [
+    "angular",
+    "bootstrap",
+    "components",
+    "accordion",
+    "alert",
+    "buttons",
+    "carousel",
+    "collapse",
+    "datepicker",
+    "dropdown",
+    "modal",
+    "pagination",
+    "popover",
+    "progressbar",
+    "rating",
+    "tabset",
+    "timepicker",
+    "tooltip",
+    "typeahead"
+  ],
+  "author": "Pawel Kozlowski <pkozlowski.opensource@gmail.com>",
+  "homepage": "https://ng-bootstrap.github.io",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ng-bootstrap/ng-bootstrap.git"
+  },
+  "npmName": "@ng-bootstrap/ng-bootstrap",
+  "npmFileMap": [
+    {
+      "basePath": "bundles",
+      "files": [
+        "ng-bootstrap*.+(js|map)"
```

Comment:
I think `ng-bootstrap*` is good for here.
Because bundle file is a set of files of the same type to limit the number and the amount the data to be downloaded by the browser.
So, the files under `bundles` can be used by the web.

However, I'll respect maintainer's decision.

Reply:
Yeah I think @sufuf3 is right from her experience, but as she also approved this PR, and the pattern looks okay in this case, I'll go to merge this one, feel free to send another PR to approve it, or I think @sufuf3 can request for change next time ;)